frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/ErrorMessages.java

```diff
@@ -68,6 +69,9 @@ public class ErrorMessages {
     public static final String OBLIGATION_NOT_UPDATED = "Obligation could not be updated.";
     public static final String VENDOR_DUPLICATE = "A vendor with the same name already exists.";
     public static final String ERROR_VENDOR = "Error: Invalid vendor Name or Url.";
+    public static final String ERROR_BULK_DELITING = "Error while bulk deleting";
+    public static final String ERROR_BULK_DELITING_IN_BACKEND = "Error while bulk deleting in backend.";
```

Comment:
Make it ERROR_BULK_DELETING and ERROR_BULK_DELETING_IN_BACKEND

Reply:
done